testdata/translator/rss/feed_image_-_rss_description.xml

```diff
@@ -0,0 +1,8 @@
+<!--
+Description: channel image
+-->
+<rss version="0.91">
+  <channel>
+    <description><img src="http://example.com/description.png"></description>
```

Comment:
Description is parsed as `"<img src=\"http://example.com/description.png\"></description>"` which doesn't make any sense to me. Is this a bug or expected?

Reply:
Here's the test output:

```
--- FAIL: TestDefaultRSSTranslator_Translate (0.10s)
    translator_test.go:45: 
                Error Trace:    /home/joe/gofeed/translator_test.go:45
                Error:          Not equal: 
                                expected: &gofeed.Feed{Title:"", Description:"<img src=\"http://example.com/description.png\">", Link:"", FeedLink:"", Links:[]string(nil), Updated:"", UpdatedParsed:<nil>, Published:"", PublishedParsed:<nil>, Author:(*gofeed.Person)(nil), Authors:[]*gofeed.Person(nil), Language:"", Image:(*gofeed.Image)(0xc0004dc440), Copyright:"", Generator:"", Categories:[]string(nil), DublinCoreExt:(*ext.DublinCoreExtension)(nil), ITunesExt:(*ext.ITunesFeedExtension)(nil), Extensions:ext.Extensions(nil), Custom:map[string]string(nil), Items:[]*gofeed.Item{}, FeedType:"rss", FeedVersion:"0.91"}
                                actual  : &gofeed.Feed{Title:"", Description:"<img src=\"http://example.com/description.png\"></description>", Link:"", FeedLink:"", Links:[]string(nil), Updated:"", UpdatedParsed:<nil>, Published:"", PublishedParsed:<nil>, Author:(*gofeed.Person)(nil), Authors:[]*gofeed.Person(nil), Language:"", Image:(*gofeed.Image)(0xc0004dc420), Copyright:"", Generator:"", Categories:[]string(nil), DublinCoreExt:(*ext.DublinCoreExtension)(nil), ITunesExt:(*ext.ITunesFeedExtension)(nil), Extensions:ext.Extensions(nil), Custom:map[string]string(nil), Items:[]*gofeed.Item{}, FeedType:"rss", FeedVersion:"0.91"}
                            
                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -2,3 +2,3 @@
                                  Title: (string) "",
                                - Description: (string) (len=46) "<img src=\"http://example.com/description.png\">",
                                + Description: (string) (len=60) "<img src=\"http://example.com/description.png\"></description>",
                                  Link: (string) "",
                Test:           TestDefaultRSSTranslator_Translate
                Messages:       Feed file feed_image_-_rss_channel_description.xml did not match expected output feed_image_-_rss_channel_description.json
    translator_test.go:45: 
                Error Trace:    /home/joe/gofeed/translator_test.go:45
                Error:          Not equal: 
                                expected: &gofeed.Feed{Title:"", Description:"", Link:"", FeedLink:"", Links:[]string(nil), Updated:"", UpdatedParsed:<nil>, Published:"", PublishedParsed:<nil>, Author:(*gofeed.Person)(nil), Authors:[]*go
```

There appears to be an issue with the parser including the closing `</description>` tag as part of the description text.

Reply:
This seems like a bug.  I'm guessing the pull parser is thrown off by the "self closing" HTML style img tag.  I bet it doesn't mess up if it is `<img src="http://example.com/description.png" />` ?

I haven't looked yet -- but I suspect we are using some convenience method like "ParseText()` function which is supposed to consume the contents of the description tag.  That might be where we have to look.

Reply:
You're right, closing the `<img />` tag fixed the parsing behavior.

Reply:
Here? https://github.com/mmcdole/gofeed/blob/master/rss/parser.go#L155-L160

Reply:
So maybe a raw self closing img tag is not a very good test. Looking at the fivethirtyeight.com feed, the description sometimes has html syntax, but rss is an xml format so the description is formatted in a `<![CDATA[...]]>` text block. I think tests that have html content should be CDATA-encoded.